src/styles/index.scss

```diff
@@ -1,3 +1,89 @@
+@import './utils/variables';
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+  border: $border-width dotted $color-border;
+  border-radius: 50%;
+
+  &__center {
+    position: absolute;
+    width: $center-diameter;
+    height: $center-diameter;
+    background-color: $color-center;
+    border-radius: 50%;
+    z-index: 1;
+  }
+
+  &__hand {
+    position: absolute;
+
+    &--minutes {
+      width: $minute-hand-width;
+      height: $minute-hand-length;
+      top: 20vmin;
+
+      background-color: $color-minute-hand;
+
+      animation-name: rotate-minutes;
+      animation-duration: 3600s;
+      animation-iteration-count: infinite;
+      animation-timing-function: steps(60);
```

Comment:
The 'animation-timing-function' set to 'steps(60)' for the minute hand might not provide a smooth animation. If a smooth rotation is desired, consider using a different timing function, such as 'linear' or a 'cubic-bezier' curve.